contents/data-structure/code/Tree/Tree.ts

```diff
@@ -0,0 +1,64 @@
+class TreeNode {
+  parent: TreeNode | null;
+
+  children: TreeNode[];
+
+  data: any;
+
+  level: number;
+
+  constructor(key: any, level: number, parent: TreeNode | null) {
+    this.parent = parent;
+    this.data = key;
+    this.children = [];
+    this.level = level;
+  }
+
+  addChlid(key: any): void {
+    this.children.push(new TreeNode(key, this.level + 1, this));
+  }
+
+  printChildNodes(): void {
+    this.children.forEach((e) => console.log(e.data));
+  }
```

Comment:
```suggestion
  printChildNodes(): void {
    this.children.forEach((child) => console.log(child.data));
  }
```
`child` 등으로 명시적인 네이밍을 사용하는 것이 더 좋지 않을까요?

Reply:
저거 하면서 졸렸나봐요 ... 밑에는 child로 했는데!.. 수정했습니다 👍